bundle/manifests/limitador-operator.clusterserviceversion.yaml

```
@@ -234,7 +225,7 @@ spec:
   - email: didier@redhat.com
     name: Didier Di Cesare
   maturity: alpha
-  minKubeVersion: 1.8.0
+  minKubeVersion: 1.26.0
```

I think this change in version prevents install on Openshift 4.12 (Kubernetes 1.25) so might be better setting to 1.25.0 for now
```suggestion
  minKubeVersion: 1.25.0
```

---

Does this PR depend on functionality in a new Kubernetes?

---

[Validation Rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) in beta in k8s 1.25

Using validation rules is not mandatory for implementing the feature of this PR. But saves us from implementing in the controller and adding validation errors in the status. Furthermore, the UX is much better with the validation rules. The API server will reject any create/update operation if the validation fails.

---

In that case we need to remember this requirement when doing OperatorHub release as it limits OpenShift versions supported